gen3-integration-tests/tests/test_audit_service.py

```diff
@@ -343,3 +349,50 @@ def test_audit_ras_login_events(self, page: Page):
         assert audit.check_query_results(
             "login", "smarty_two", params, expected_results
         )
+
+    @pytest.mark.skipif(
+        "nightly-build" not in pytest.hostname
+        and os.getenv("GEN3_INSTANCE_TYPE") == "HELM_LOCAL",
+        reason="Test is being run on Helm and would run only on nightly-build",
+    )
+    def test_audit_oidc_fence_client_login_events(self, page: Page):
+        """
+        Scenario: Perform login in via the OIDC flow (IDP RAS)
+        Steps :
+            1. Login with a fence client using OIDC flow (IDP RAS)
+            2. Call Audit log API using auxAcct2 user
+            3. Check if entry for ORCID user is present
+        NOTE : This test requires CI_TEST_ORCID_ID & CI_TEST_ORCID_PASSWORD
+        secrets to be configured with ORCID credentials
+        """
+        client_id = pytest.clients["ras-test-client"]["client_id"]
+        scope = "openid user data google_credentials"
+        username = os.environ["CI_TEST_RAS_EMAIL"].split("@")[0]
+        password = os.environ["CI_TEST_RAS_PASSWORD"]
+        email = os.environ["CI_TEST_RAS_EMAIL"]
+        audit = Audit()
+        ras = RAS()
+        timestamp = math.floor(time.mktime(datetime.datetime.now().timetuple()))
+        params = [
+            "start={}".format(timestamp),
+        ]
+        token = ras.get_auth_code(
+            scope=scope,
+            username=username,
+            password=password,
+            client_id=client_id,
+            email=email,
+            page=page,
+        )
+        assert token != "", "Token returned was empty"
+
+        # Check the query results with auxAcct2 user
+        expected_results = {
+            "username": username,
+            "idp": "ras",
+            "client_id": client_id,
+            "status_code": 302,
+        }
+        assert audit.check_query_results(
+            "login", "smarty_two", params, expected_results
```

Comment:
the username is hardcoded to "smarty_two" in all these tests, can we use the "username" variable instead?